recipe_scrapers/tests/test_bettycrocker.py

```diff
@@ -0,0 +1,66 @@
+import os
+import unittest
+
+from recipe_scrapers.bettycrocker import BettyCrocker
+
+
+class TestAllRecipesScraper(unittest.TestCase):
```

Comment:
Fantastic! Quick learner 😉 This here has to be `class TestBettyCrocker`. Everything else is perfect! 🍻 🍻

Reply:
Sorry about.
I'll fix it now and re-submit.

Thank you.  

Just an FYI, I have one more ready to go through the PR process, probably today.  I have some other ideas that I'll work on implementing, probably one or two others within the week.